ida/data.yml

```
@@ -10089,12 +10092,90 @@ classes:
     funcs:
       0x1414B1DE0: ctor
       0x1414BB840: HandleEnterContentInfoPacket  # static
+  Client::Game::InstanceContent::PublicContentBonding:
+    vtbls:
+      - ea: 0x141C30BC8
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B2DF0: ctor
+  Client::Game::InstanceContent::PublicContentTripleTriad:
+    vtbls:
+      - ea: 0x141C31748
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B3370: ctor
+  Client::Game::InstanceContent::PublicContentEureka:
+    vtbls:
+      - ea: 0x141C322C8
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B4550: ctor
+  Client::Game::InstanceContent::PublicContentRising:
+    vtbls:
+      - ea: 0x141C33A28
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B6090: ctor
+  Client::Game::InstanceContent::PublicContentLeapOfFaith:
+    vtbls:
+      - ea: 0x141C345A8
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B6970: ctor
+  Client::Game::InstanceContent::PublicContentEurekaHydatos:
+    vtbls:
+      - ea: 0x141C32E78
+        base: Client::Game::InstanceContent::PublicContentEureka
+    funcs:
+      0x1414B5C00: ctor
+  Client::Game::InstanceContent::PublicContentDiadem:
+    vtbls:
+      - ea: 0x141C35128
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B6D70: ctor
+  Client::Game::InstanceContent::PuclicContentBozja:
+    vtbls:
+      - ea: 0x141C36828
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B8A00: ctor
+  Client::Game::InstanceContent::PuclicContentSoutherFront:
+    vtbls:
+      - ea: 0x141C373D0
+        base: Client::Game::InstanceContent::PuclicContentBozja
+    funcs:
+      0x1414B9210: ctor
+  Client::Game::InstanceContent::PublicContentDelubrum:
+    vtbls:
+      - ea: 0x141C396D0
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B96B0: ctor
+  Client::Game::InstanceContent::PublicContentDelubrumSavage:
+    vtbls:
+      - ea: 0x141C3A280
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B9740: ctor
+  Client::Game::InstanceContent::PuclicContentZadnor:
+    vtbls:
+      - ea: 0x141C37F78
+        base: Client::Game::InstanceContent::PuclicContentBozja
```

Puclic -> Public, now as suggestion.^^

```suggestion
  Client::Game::InstanceContent::PublicContentBozja:
    vtbls:
      - ea: 0x141C36828
        base: Client::Game::InstanceContent::PublicContentDirector
    funcs:
      0x1414B8A00: ctor
  Client::Game::InstanceContent::PublicContentSoutherFront:
    vtbls:
      - ea: 0x141C373D0
        base: Client::Game::InstanceContent::PublicContentBozja
    funcs:
      0x1414B9210: ctor
  Client::Game::InstanceContent::PublicContentDelubrum:
    vtbls:
      - ea: 0x141C396D0
        base: Client::Game::InstanceContent::PublicContentDirector
    funcs:
      0x1414B96B0: ctor
  Client::Game::InstanceContent::PublicContentDelubrumSavage:
    vtbls:
      - ea: 0x141C3A280
        base: Client::Game::InstanceContent::PublicContentDirector
    funcs:
      0x1414B9740: ctor
  Client::Game::InstanceContent::PublicContentZadnor:
    vtbls:
      - ea: 0x141C37F78
        base: Client::Game::InstanceContent::PublicContentBozja
```